code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -256,11 +256,10 @@
 			dismantle_wall()
 
 /turf/closed/wall/r_wall/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
-	if(!the_rcd.canRturf)
-		return FALSE
-	return ..()
+	if(istype(the_rcd, /obj/item/construction/rcd/borg) || istype(the_rcd, /obj/item/construction/rcd/combat))
```

Comment:
There is almost certainly a cleaner way to do this but I just wanted to get it to stop asking for a var when it wasnt really nessecary anyhow. If a maintainer asks me to make it borg only again I will do so.

Reply:
What's wrong with a var? A snowflake check is just worse

Reply:
why did you think this was a good idea? please use the var

Reply:
Okay its just feels bad